.env.example

```diff
@@ -1 +1 @@
-JOURNAL_API_URL=
\ No newline at end of file
+JOURNAL_API_URL=apiStringOrLocalJsonString
```

Comment:
does this get parsed?

Reply:
It's .env.example so it should be ignored...